src/index.js

```diff
@@ -236,7 +236,7 @@ class AvatarEditor extends React.Component {
     }
 
     // don't paint a border here, as it is the resulting image
-    this.paintImage(canvas.getContext('2d'), this.state.image, 0)
+    this.paintImage(canvas.getContext('2d'), this.state.image, 0, 1)
```

Comment:
Because scaling is now built into `paintImage`, we need to override it here so that getImageScaledToCanvas doesn't start outputting an image of a different size.  We could consider whether the image *should* be larger for HiDPI devices, but I'd suggest we think about that as a separate change.